packages/block-library/src/read-more/index.js

```diff
@@ -16,6 +17,11 @@ export { metadata, name };
 export const settings = {
 	icon,
 	edit,
+	example: {
+		attributes: {
+			content: _x( 'Read More', 'Example text for the Read More block' ),
+		},
```

Comment:
```suggestion
		attributes: {
			content: __( 'Read more' ),
		},
```

- Sorry, context may not be necessary. This text already exists, so it would save the translator the effort of translating it anew.
- To ensure an exact match with [the default text](https://github.com/WordPress/gutenberg/blob/b4304f8bf6bd9b890b4108adcc326cd586a3ab4e/packages/block-library/src/read-more/edit.js#L39), sentence case is preferred over title case.

Reply:
Thanks for the suggestion @t-hamano. I've implemented these changes in the latest commit 🚀.